dace/symbolic.py

```diff
@@ -820,6 +829,11 @@ def _print_Infinity(self, expr):
     def _print_NegativeInfinity(self, expr):
         return '-INFINITY'
 
+    def _print_Symbol(self, expr):
+        if expr.name == 'NoneSymbol':
+            return 'None'
```

Comment:
Shouldn't this be `NULL` or `nullptr`? The description of the class is: `Several notational corrections for integer math and C++ translation`.